com.ibm.streamsx.inet/impl/java/src/com/ibm/streamsx/inet/http/HTTPPostOper.java

```diff
@@ -182,7 +184,17 @@ public void setExtraHeaders(List<String> val) {
 	public void setAcceptAllCertificates(boolean val) {
 		this.acceptAllCertificates = val;
 	}
-	
+	@Parameter(optional=true, 
```

Comment:
Actually I don't think a set is needed, if `postAttributes` is set then just do something like:

```
    for (TupleAttribute<Tuple, ?> postAttr : postAtttributes) {
            String attrName = postAttr.getAttribute().geName();
            String attrValue = tuple.getString(postAttr.getAttribute().getIndex());
            // add attrName,  attrValue  into post request
        }
```